app/controllers/oauth2_controller.rb

```diff
@@ -5,6 +5,9 @@
 # @!visibility private
 class Oauth2Controller < ApplicationController
 
+  # Default to forcing SSL unless disabled
+  force_ssl if: -> { RailsApiAuth.force_ssl.nil? || RailsApiAuth.force_ssl }
```

Comment:
I don't think the `nil` check is necessary here. Instead just use `!!RailsApiAuth.force_ssl` so that nil evaluates to `false`.